packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -101,27 +110,48 @@ async function main() {
   const proposerV2 = await proposerFactory.deploy(votingToken.address, defaultBond, governorV2.address, finder.address);
   console.log("Deployed ProposerV2: ", proposerV2.address);
 
-  console.log("6. Set ProposerV2 as the proposer of GovernorV2");
+  console.log("6. Deploying EmergencyProposer");
+  const quorum = hre.web3.utils.toWei("10000000", "ether");
+  // In localhost network we allow to not set the emergency executor address and default to the first account.
+  if (!process.env[EMERGENCY_EXECUTOR] && hre.network.name != "localhost") throw new Error("No emergency executor set");
```

Comment:
Is this script used for production deployments? Or just for simulations/tests?

If just for tests, could we just default to the first account for all networks?

If used for production deployments, I agree that we would want to avoid _accidentally_ setting that value to the deployer account.

Reply:
The goal is to use the scripts for the production deployment, that's why I'm adding extra checks and every test/fork related functionality must be enabled with en env var like `TEST_DOWNGRADE` or `EMERGENCY_PROPOSAL` to make it work.